internal/archive/extract.go

```diff
@@ -97,12 +99,19 @@ func FetchAndExtract(log logr.Logger, ctx context.Context, url string, timeout t
 	}, nil
 }
 
+func retryable(err *url.Error) bool {
+	// If we get any sort of operational error before an HTTP response we retry it.
+	var opError *net.OpError
+	return err.Timeout() || err.Temporary() || errors.As(err, &opError)
```

Comment:
Can you use `errors.Is` as the last condition since you really only want to assert the type and not extract the error?

Reply:
Apparently, no, that's a value check against the exact error.